backend/gzdb/db.go

```diff
@@ -26,17 +26,18 @@ type User struct {
 	Plan        string `gorm:"default:'free'"`
 }
 
-type UserRole struct {
-	// gorm.Model without ID
+type MembershipRole struct {
 	CreatedAt time.Time
 	UpdatedAt time.Time
 	DeletedAt gorm.DeletedAt `gorm:"index"`
 
-	UserID  uint `gorm:"primaryKey;autoIncrement:false"`
-	User    User
-	OrgType string `gorm:"primaryKey"` // "event" or "community"
-	OrgID   uint   `gorm:"primaryKey;autoIncrement:false"`
-	Role    string `gorm:"primaryKey"`
+	OrgChildType string `gorm:"primaryKey"` // e.g. "user", "event"
```

Comment:
use EntityType & EntityID

and OrgType & OrgID

Reply:
[c617cca](https://github.com/samouraiworld/zenao/pull/613/commits/c617cca4996b598306b82b44695bc338c11877da)